src/main/resources/doc/fr/html/guide/prefs/pref-intl.html

```diff
@@ -61,7 +61,7 @@ <h1>
             </table>
           </center>
           <p>
-            Selon les continents un style à la préférence. "IEC" est plus populaire aux Etats-Unis et ANSI est préféré en Europe. Certaines personnes se réfèrent à ces styles selon ces régions.
+            Selon les continents un style à la préférence. <i>IEC</i> est plus populaire aux Etats-Unis et <i>ANSI</i> est préféré en Europe. Certaines personnes se réfèrent à ces styles selon ces régions.
```

Comment:
See above: "IEC" and "ANSI" are mixed up!

Reply:
As before